apps/ember-test-app/tests/fastboot/image-test.js

```diff
@@ -13,7 +13,7 @@ module('FastBoot | image', function (hooks) {
     assert.dom('img[data-test-simple-image]').exists();
     assert
       .dom('img[data-test-simple-image]')
-      .hasAttribute('src', /assets\/aurora-3840w(-\w+)?\.jpg/);
+      .hasAttribute('src', /assets\/aurora-1920w-(-\w+)?\.jpg/);
```

Comment:
This test result surprised me, I'd expect it to generate 1280 same as the rest. It's somewhat close to the configured value, but should be looked into before a merge.